app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :user, class_name: 'User', foreign_key: :user_id
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- You have done well so far, but just as you did for the `post` and `comment` model, please ensure the likes_counter is updated when a model is created and saved in the database. Please fix this 🙏

Reply:
Thanks a lot @mckent05 for identifying the missing method call.

I have now updated the class:
![Update_like_counter](https://github.com/Zafron047/Blog_app_by_Rails/assets/117600135/01da21b1-bcf9-41fe-a597-aaccc21133c1)